scripts/setup-centos9.sh

```diff
@@ -229,11 +246,21 @@ function install_velox_deps {
     else
       echo "Skipping installation of build dependencies since INSTALL_PREREQUISITES is not set"
     fi
-    # Activate gcc12; enable errors on unset variables afterwards.
-    source /opt/rh/gcc-toolset-12/enable || exit 1
-    set -u
+    if [[ ${USE_CLANG} != "false" ]]; then
+      export CC=/usr/bin/clang-15
+      export CXX=/usr/bin/clang++-15
+    else
+      # Activate gcc12; enable errors on unset variables afterwards.
+      source /opt/rh/gcc-toolset-12/enable || exit 1
+      set -u
+    fi
     install_velox_deps
     echo "All dependencies for Velox installed!"
+    if [[ ${USE_CLANG} != "false" ]]; then
```

Comment:
Do we need this if block?

Reply:
I wanted to let the user know how to use it with the actual make command. So far they would have only set USE_CLANG env variable but not set anything else. Unless we assume they set the CC/CXX from the very beginning.

Reply:
Makes sense!